packages/cbioportal-utils/src/model/Uniprot.ts

```diff
@@ -0,0 +1,31 @@
+// Temporary model for Uniprot data
```

Comment:
We will remove this file once we merge genome-nexus/genome-nexus#442

Reply:
I think eventually we want to get Uniprot features from Genome Nexus instead of directly using Uniprot API. genome-nexus/genome-nexus#442 does not implement a cache yet so I guess we can update this part later.